Parent/Parent/Create Account/CreateAccountViewController.swift

```diff
@@ -0,0 +1,185 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2020-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import UIKit
+import Core
+
+class CreateAccountViewController: UIViewController, ErrorViewController {
+    @IBOutlet weak var scrollView: UIScrollView!
+    @IBOutlet weak var stackView: UIStackView!
+    @IBOutlet weak var name: CreateAccountRow!
+    @IBOutlet weak var email: CreateAccountRow!
+    @IBOutlet weak var password: CreateAccountRow!
+    @IBOutlet weak var createAccountButton: DynamicButton!
+    @IBOutlet weak var termsAndConditionsLabel: DynamicLabel!
+    @IBOutlet weak var alreadyHaveAccountLabel: DynamicLabel!
+    var keyboardSpace: NSLayoutConstraint = NSLayoutConstraint()
```

Comment:
this is usually the scrollView.bottomAnchor to safeArea.bottomAnchor. I think this is making the scrollView not shrink when the keyboard appears, so you can't easily scroll to the next form field.